src/clue.js

```diff
@@ -2,25 +2,78 @@
 
 // Suspects Array
 
-const suspectsArray = [];
+const suspectsArray = [
+    {firstName: "", lastName:"", occupation:"", age: 30, description:"", image:"", color:""},
+    {firstName: "", lastName:"", occupation:"", age: 30, description:"", image:"", color:""},
+    {firstName: "", lastName:"", occupation:"", age: 30, description:"", image:"", color:""},
+    {firstName: "", lastName:"", occupation:"", age: 30, description:"", image:"", color:""},
+    {firstName: "", lastName:"", occupation:"", age: 30, description:"", image:"", color:""},
+    {firstName: "", lastName:"", occupation:"", age: 30, description:"", image:"", color:""}
+];
 
 // Rooms Array
 
-const roomsArray = [];
+const roomsArray = [
+    {room1:"", name:""},
+    {room2:"", name:""},
+    {room3:"", name:""},
+    {room4:"", name:""},
+    {room5:"", name:""},
+    {room6:"", name:""},
+    {room7:"", name:""},
+    {room8:"", name:""},
+    {room9:"", name:""},
+    {room10:"", name:""},
+    {room11:"", name:""},
+    {room12:"", name:""},
+    {room13:"", name:""},
+    {room14:"", name:""},
+    {room15:"", name:""}
+
+]
 
 // Weapons Array
 
-const weaponsArray = [];
+const weaponsArray = [
+    {weapon1:"", name:"", weight: 2.2},
+    {weapon2:"", name:"", weight: 2.2},
+    {weapon3:"", name:"", weight: 2.2},
+    {weapon4:"", name:"", weight: 2.2},
+    {weapon5:"", name:"", weight: 2.2},
+    {weapon6:"", name:"", weight: 2.2},
+    {weapon7:"", name:"", weight: 2.2},
+    {weapon8:"", name:"", weight: 2.2},
+    {weapon9:"", name:"", weight: 2.2}    
+];
 
 
 // ITERATION 2
 
-function selectRandom() {}
+function selectRandom(arr) {
+    if (arr.length === 0){
+        return undefined;
+    } else if (arr.length === 1){
+        return arr[0];
+    } else {
+        const randomIndex = Math.floor(Math.random() * arr.length);
+        return arr[randomIndex];
```

Comment:
This is correct but actually there is no need for the conditions here, you can just do:

```
function selectRandom(arr) {
           const randomIndex = Math.floor(Math.random() * arr.length);
           return arr[randomIndex];
}
```

Because if the array is empty `randomIndex` will be equal to `0` and since there is no index 0 `[0]` it will already return undefined. And if we only have 1 element inside the array `randomIndex` will be `0` again but this time we have the index 0 `[0]` so we will return it.